README.md

```diff
@@ -13,7 +13,308 @@ docker-compose up
 ```
 
 # Business logic interface
-TBA
+
+## Business Logic Setup
```

Comment:
We don't need this section for this repo right?

Reply:
I suppose we don't technically /need/ it, but I don't think it will hurt.  Are we planning on giving this to people to use /without/ spotlight?